docs/AWS_Integration_Guide/Configuring_AWS_integration.md

```diff
@@ -96,7 +96,7 @@ https://coiote-aws-int-[REGION-NAME].s3.[REGION-NAME].amazonaws.com/coiote-aws-i
 
     - **coioteDMrestUsername** - username of the created CoioteDM account.
     - **coioteDMrestPassword** - password of the created CoioteDM account.
-    - **coioteDMrestUri** - URL address and port of your Coiote DM installation.
+    - **coioteDMrestUri** - URL address and port of your Coiote DM installation. In most cases the port should be `8088`.
```

Comment:
I'd reword it to inform about possible explicitly:
`URL address and port of your Coiote DM installation. The port should always be `8088`, unless you are using `lwm2m-test.avsystem.io` - there the port part can be skipped.`